MoleculeVAE-reImplementation/train.py

```diff
@@ -167,30 +169,33 @@ def load_data_by_task(params):
             -test_dataset,
         '''
     if params['do_prop_pred']:
-        X_train, X_test, Y_train, Y_test, character_index_lookup_dict = load_dataset(params)
+        X_train, X_test, Y_train, Y_test, _ = load_dataset(params)
     else:
-        X_train, X_test, character_index_lookup_dict = load_dataset(params)
+        X_train, X_test, _ = load_dataset(params)
 
     train_dataset = CustomMoleculeDataset(X_train, Y_train)
-    train_set_final, valid_set_final = split_validation_dataset(train_dataset, percentage_train=0.8)
+    train_dataset, valid_dataset = split_validation_dataset(
+        train_dataset, percentage_train=0.8)
     test_dataset = CustomMoleculeDataset(X_test, Y_test)
-    test_set_final, _ = split_validation_dataset(test_dataset, percentage_train=1)# To control the number of percentage
-    training_data_dic = pd.DataFrame(list(zip( train_set_final, valid_set_final, test_dataset )),
-                      columns=['train_set_final', 'valid_set_final', 'test_dataset'])
-    return training_data_dic
+
+    training_data_dict = dict(
+        train_set_final=train_dataset,
+        valid_set_final=valid_dataset,
+        test_dataset=test_dataset)
+    return training_data_dict
 
 
 
 '''Loss function '''
 
-def vae_loss(x_decoded_mean, x, z_mean, z_logvar):
```

Comment:
`vae_loss` is here used as a function name but in other places is used as a variable name. This leads my IDE to get confused and possibly also the code (though I can't verify that). 
Confusions like this are part of why it's good to keep variables as nouns and functions as verbs. This is also what's recommended by PEP.